drivers/entropy/entropy_stm32.c

```diff
@@ -476,39 +457,33 @@ static int entropy_stm32_rng_init(const struct device *dev)
 
 	dev_data->clock = DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE);
 
-	res = clock_control_on(dev_data->clock,
-		(clock_control_subsys_t *)&dev_cfg->pclken);
+	res = clock_control_on(dev_data->clock, (clock_control_subsys_t *)&dev_cfg->pclken);
 	__ASSERT_NO_MSG(res == 0);
 
-
-#if DT_INST_NODE_HAS_PROP(0, health-test-config)
-#if DT_INST_NODE_HAS_PROP(0, health-test-magic)
+#if DT_INST_NODE_HAS_PROP(0, health - test - config)
+#if DT_INST_NODE_HAS_PROP(0, health - test - magic)
```

Comment:
Seems clang formatting is broken.

Reply:
oh, good catch :-o

Reply:
euh, I think this is an error as well right? all properties are converted from - to _ right?

Reply:
@erwango 
just tested it, and never worked afaik. I can fix them properly with the '_', but probly this needs a new PR?

Reply:
ok, txs. Yes a new PR (+issue) would be great so we can put this in DV3.0.

Reply:
done in #42386

Reply:
reverted this change, and let #42386 fix this

Reply:
Actually, I think these DT macros are broken, they should be using `_`:

```
#if DT_INST_NODE_HAS_PROP(0, health_test_config)
```

Reply:
yes, see previous comments. I've fixed them in #42386